src/file-picker/src/FilePicker.js

```diff
@@ -157,4 +164,17 @@ export default class FilePicker extends PureComponent {
   handleButtonClick = () => {
     this.fileInput.click()
   }
+
+  handleBlur = e => {
+    const { onBlur } = this.props
+    const { files } = this.state
+
+    // Redux-form derives field value from bubbled onBlur event, which is fake here
+    if (e && e.stopPropagation) e.stopPropagation()
+    if (this.fileInput) this.fileInput.blur()
+
+    if (onBlur) {
+      onBlur(files)
```

Comment:
First argument in onBlur is the same as in onChange - requirement of redux-form, if event with data is not available.

Reply:
We do have a `safeInvoke` function we could use instead of this `if` check.